ow_lander/launch/ros_controllers.launch

```diff
@@ -5,10 +5,13 @@
   <rosparam file="$(find ow_lander)/config/ros_controllers.yaml" command="load"/>
 
   <!-- Load the controllers -->
-  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
-    output="screen" args="arm_controller antenna_controller"/>
-
   <node name="joint_controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
 		output="screen" args="joint_state_controller" />
 
+  <node name="controller_spawner_active" pkg="controller_manager" type="spawner" respawn="false"
+    output="screen" args="arm_controller antenna_controller"/>
+
+  <node name="controller_spawner_stopped" pkg="controller_manager" type="spawner" respawn="false"
```

Comment:
It's unclear to me what each controller is for based on the names and arguments. Some descriptive comments would be good.

Reply:
Addressed